src/utils/constants.ts

```diff
@@ -157,6 +157,11 @@ enum RegionMapping {
   'north-america' = 5,
 }
 
+enum WalletModeMapping {
+  noUI = 0,
+  UI = 1,
+}
```

Comment:
Call it just `WalletMode`?

```suggestion
enum WalletMode {
  noUI = 0,
  UI = 1,
}
```